cypress/e2e/HomePage/delete.project.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Test homepage: project deletion
+
+  Scenario: Delete existing project
```

Comment:
Add new e2e test to delete project and recreate same project

Reply:
done